MergerCli/SourceParser.cs

```diff
@@ -79,14 +90,47 @@ private IData ParseFileSource(string[] args, ref int idx, int batchSize, bool is
             return this._dataFactory.CreateDatasource(sourceType, sourcePath, batchSize, isOneXOne, origin, isBase);
         }
 
+        private IData ParseGpkgSource(string[] args, ref int idx, int batchSize, bool isBase)
+        {
+            const int requiredParamCount = 2;
+            const int optionalParamCount = 4;
+            int paramCount = this.ValidateAndGetSourceLength(args, idx, requiredParamCount, optionalParamCount);
+            string sourceType = args[idx];
+            string sourcePath = args[idx + 1];
+            bool isOneXOne = false;
+            GridOrigin? origin = null;
+            Extent? extent = null;
+            int? maxZoom = null;
```

Comment:
Add comment - this affects only the `gpkg_tile_matrix` table.

Reply:
done